pkg/boot/types.go

```diff
@@ -0,0 +1,19 @@
+package boot
+
+import (
+	"net"
+	"time"
+
+	"github.com/coreos/go-etcd/etcd"
+)
+
+// Boot boot struct
```

Comment:
this comment's a little confusing. Would you mind elaborating more on how someone would use this structure? Maybe a little more clarification is required on what `Boot` actually is would be good :)